packages/cozy-client/src/CozyClient.js

```diff
@@ -1685,8 +1685,14 @@ instantiation of the client.`
       Q('io.cozy.settings').getById('io.cozy.settings.capabilities')
     )
 
+    const { data: instanceData } = await this.query(
```

Comment:
Should be getOrFechFromState (I don’t know the name exactly) and you should add a fetchPolicy

Reply:
Updated. It should be always fresh so no fetchPolicy no?

Reply:
@zatteo I think using `fetchQueryAndGetFromState` as proposed by @Crash-- is only useful if we add a fetchPolicy. That being said, I'm not sure we want this kind of logic at this level, I don't think I like the idea of a forced fetchPolicy that could lead to unexpected results from the caller.
In any case, the behaviour should be the same for the other query just above on io.cozy.settings.capabilities.

Reply:
So :
- no fetchPolicy because we need it fresh
- same method for the two calls : decided to stay on query for the moment